### PR TITLE
Feature/CORS-Middleware

### PR DESCRIPTION
**Pull request by xinruili07**
# Description

Added the CORS Middleware for backend/frontend connection.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Tested by sending a request from the frontend (localhost:8080) with a different origin than the backend (localhost:8000).

- [ ] Test A
- [ ] Test B

# Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I will rebase before merging

### CODE REVIEW

File: vendor-frontend/src/views/Dashboard.vue [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+    export default {
+        data () {
+            return {       
+                info: "",
+            }   
+        },
+        mounted(){
+            axios
+            .get("http://127.0.0.1:8000/vendors/1")
```
Comment (Ty-Won) [inline vendor-frontend/src/views/Dashboard.vue:28]: does this need to be hardcoded? Or can it be referenced somewhere?

Comment (xinruili07) [reply vendor-frontend/src/views/Dashboard.vue:28]: Oh, I can change that part, I only created this pull request to add the CORS-middlewares for George.

Comment (Ty-Won) [reply vendor-frontend/src/views/Dashboard.vue:28]: oh okay sounds good @xinruili07  if you can change it, lets do it

Comment (Ty-Won) [reply vendor-frontend/src/views/Dashboard.vue:28]: bump

**Pull request closed without merging**